CHANGELOG.md

```diff
@@ -12,6 +12,70 @@
 
 ## Improvements
 
+# v1.28.0-rc1 / 2024-07-01
+
+This is the first release candidate of the upcoming MANDATORY Lotus v1.28.0 release, which will deliver the Filecoin network version 23, codenamed Waffle.
+
+**This release does NOT set a calibration network upgrade epoch yet, which will be included in the subsequent release candidate. This release does NOT set the mainnet upgrade epoch yet, which will be updated in the final release.**
```

Comment:
```suggestion
**This release canidate does NOT set a calibration network upgrade epoch, it will be added in the second release candidate, expected to be released July 4th. This release does NOT set the mainnet upgrade epoch yet, which will be updated in the final release.**
```